examples/land_coverage_from_scaled_map.jl

```diff
@@ -0,0 +1,11 @@
+using Klimakoffer
+
+world = Klimakoffer.read_geography("../input/world/The_World128x65.dat", 128, 65)
```

Comment:
```suggestion
world = Klimakoffer.read_geography(joinpath(@__DIR__, "..","input","world","The_World128x65.dat"), 128, 65)
```
This is the right way to specify the path. Two reasons:

- The use of the slash (`/`) and backslash (`\`) symbols for paths in unix-like and windows systems: https://stackoverflow.com/questions/38428561/difference-between-forward-slash-and-backslash-in-file-path
- The relative path

Please change the other paths accordingly.